client/src/pages/HealthCheckPage/HealthCheckPage.tsx

```diff
@@ -21,7 +22,7 @@ const HealthCheckPage: React.FC = () => {
     setError('');
     
     try {
-      const response = await axios.get<HealthCheckResponse>('http://localhost:3000/api/health-check');
+      const response = await axios.get<HealthCheckResponse>(`http://localhost:3000/${API_PREFIX}/health-check`);
```

Comment:
we should move it to services, and also create domain const

Reply:
Done